DM-164: Update documentation for migrated single and bulk general dev… (#509)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/connecting-devices.md

Depending on the type of device you want to connect, not all steps of the following process may be relevant.
{{< /c8y-admon-info >}}

> 1. Click **Registration** in the **Devices** menu of the navigator.

Reviewer: @pilosag This sentence needs to stay partly since the user first has to navigate to this page.

Reviewer 2: does this refer to line 98? restored this step (it's now line 101).

Depending on the type of device you want to connect, not all steps of the following process may be relevant.
{{< /c8y-admon-info >}}

> 1. Expand device registration dropdown menu by clicking **Register device** button 

Reviewer: ```suggestion
1. Click **Register device** at the right of the top bar and from the dropdown menu select **Single registration** > **General**.
```

> 2. Find `SINGLE REGISTRATION` section in the menu and then select **General**.

Reviewer: ```suggestion
```

  <img src="/images/users-guide/DeviceManagement/devmgmt-dropdown-menu.png" alt="Bulk registration" style="max-width: 100%">

> 3. In the result, **REGISTER DEVICES** form will be displayed.

Reviewer: ```suggestion
The **Register devices** dialog box will be displayed.
```

  <img src="/images/users-guide/DeviceManagement/devmgmt-registration-general.png" alt="General device registration" style="max-width: 100%">

> 4. In the **Device ID** field, enter a unique ID for the device. To determine the ID, consult the device documentation. In case of mobile devices the ID usually is the IMEI (International Mobile Equipment Identity) often found on the back of the device.

Reviewer: ```suggestion
2. In the **Device ID** field, enter a unique ID for the device. To determine the ID, consult the device documentation. In case of mobile devices the ID usually is the IMEI (International Mobile Equipment Identity) often found on the back of the device.
```

> 5. Optionally, select a group to assign your device to after registration, see also [Grouping devices](#grouping-devices).

Reviewer: ```suggestion
3. Optionally, select a group to assign your device to after registration, see also [Grouping devices](#grouping-devices).
```

> 6. Click **Add new device** to register one more device. Again, enter the device ID and optionally select a group. This way, you can add multiple devices in one step.

Reviewer: ```suggestion
4. Click **Add new device** to register one more device. Again, enter the device ID and optionally select a group. This way, you can add multiple devices in one step.
```

> 7. Click **Next** to register your device(s).

Reviewer: ```suggestion
5. Click **Next** to register your device(s).
```

There is no restriction on the number of devices that you can bulk-register but the more devices you add the slower the creation and operation gets.
{{< /c8y-admon-info >}}

> 1. Expand device registration dropdown menu by clicking **Register device** button 

Reviewer: ```suggestion
1. Click **Register device** at the right of the top bar and from the dropdown menu select **Bulk registration** > **General**.
```

> 2. Find `BULK REGISTRATION` section in the menu and then select **General**.

Reviewer: ```suggestion
```

  <img src="/images/users-guide/DeviceManagement/devmgmt-dropdown-menu.png" alt="Bulk registration" style="max-width: 100%">

> 3. Click **+** button to select or drag-and-drop the CSV file you want to upload.

Reviewer: ```suggestion
3. Click the Plus button to select or drag-and-drop the CSV file you want to upload.
```

There is no restriction on the number of devices that you can bulk-register but the more devices you add the slower the creation and operation gets.
{{< /c8y-admon-info >}}

1. Click **Registration** in the **Devices** menu of the navigator.

> 2. Click **Register device** at the right of the top bar and from the dropdown menu select **Bulk registration** > **General**.

Reviewer: ```suggestion
2. In the **Device registration** page, click **Register device** at the right of the top bar and from the dropdown menu select **Bulk registration** > **General**.
```
